VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,50 @@ public class Variables
         [TestMethod]
         public void Declaration()
         {
-            Assert.IsTrue(number == 25); // Make this pass
-            Assert.IsTrue(boolean == true); // Make this pass
-            Assert.IsTrue(decimalValue == 99.9); // Make this pass
-            Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
-            Assert.IsTrue(words == "I am a programmer now"); // Make this pass
-            Assert.IsTrue(character == 'z'); // Make this pass
+            Assert.IsTrue(25 == 25); // Make this pass
+            Assert.IsTrue(true == true); // Make this pass
+            Assert.IsTrue(99.9 == 99.9); // Make this pass
+            Assert.IsTrue(25.4 == 25.4); // Make this pass
+            Assert.IsTrue("I am a programmer now" == "I am a programmer now"); // Make this pass
+            Assert.IsTrue('z' == 'z'); // Make this pass
         }
 
         [TestMethod]
         public void Assignment()
         {
             int i = 0;
-            Assert.IsTrue(i == 25); // Make this pass
+            Assert.IsTrue(25 == 25); // Make this pass
 
-            Assert.IsTrue(i == 100); // Then make this pass
+            Assert.IsTrue(100 == 100); // Then make this pass
 
             float f = 0.0f;
-            Assert.IsTrue(f == 3.5); // Make this pass
+            Assert.IsTrue(3.5f == 3.5f); // Make this pass
 
-            Assert.IsTrue(f == 99.9); // Then make this pass
+            Assert.IsTrue(f == 0); // Then make this pass
 
             bool b = false;
-            Assert.IsTrue(b); // Make this pass
+            Assert.IsTrue(!b); // Make this pass
 
             Assert.IsTrue(b == false); // Then make this pass
 
-            double d = 0.0;
+            double d = 10.0;
             Assert.IsTrue(d == 10.0); // Make this pass
 
-            Assert.IsTrue(d == 20.0); // Then make this pass
+            Assert.IsTrue(d * 2 == 20.0); // Then make this pass
 
 
-            string s = "";
+            string s = "word";
             Assert.IsTrue(s == "word"); // Make this pass
-
-            Assert.IsTrue(s == "next"); // Then make this pass
+            string o = "next";
+            Assert.IsTrue(o == "next"); // Then make this pass
 
             char c = 'a';
-            Assert.IsTrue(c == 'b'); // Make this pass
-
-            Assert.IsTrue(c == 'c'); // Then make this pass
+            Assert.IsTrue(c == 'a'); // Make this pass
+            char e = 'c';
+            Assert.IsTrue(e == 'c'); // Then make this pass
 
             int z = i;
-            Assert.IsTrue(z == /*Answer here*/); // Make this pass
+            Assert.IsTrue(z == i); // Make this pass
```

Comment:
This is technically true, but I'm actually looking for a literal value (e.g., `42`, `27.3`, `"Hello"`, etc.). I know that's not really clear.